### ISSUE TITLE
VPR Noxious Gnash not being tracked via Display Permanently

### ISSUE THREAD

**Issue by Squishums** [GoldenChrysus/ffxiv-ember-overlay#54]
**Describe the bug**
The VPR debuff Noxious Gnash timer does not display when not applied but Display Permanently is ticked. It tracks the debuff correctly while it's applied. RPR's Death's Design works as expected.

**Reproduction steps**
Exact steps, both in Ember and within FFXIV, to reproduce the behavior:
1. Track Noxious Gnash Debuff
2. Click Display Permanently for Debuffs
3. Enter combat with VPR.
4. No Noxious Gnash timer
5. Apply Noxious Gnash, wait for timer to run out.
6. Noxious Gnash disappears again.

**Settings data**
```
"{\"intrinsic\":{\"table_type\":\"dps\",\"collapsed\":true,\"current_version\":\"1.9.2\",\"last_version\":\"1.9.1\",\"player_blur\":false},\"interface\":{\"player_name\":\"YOU\",\"opacity\":100,\"zoom\":100,\"text_scale\":100,\"top_right_rank\":true,\"blur_job_icons\":false,\"collapse_down\":true,\"theme\":\"ffxiv-dark\",\"minimal_theme\":true,\"horizontal\":false,\"horizontal_shrink\":false,\"horizontal_alignment\":\"left\",\"footer_when_collapsed\":false,\"footer_dps\":true,\"hide_top_bar\":true,\"decimal_accuracy\":null,\"shorten_thousands\":false,\"language\":\"en\",\"auto_hide\":\"disabled\",\"auto_hide_delay\":300,\"display_job_names\":false},\"custom\":{\"css\":null,\"metric_names\":{}},\"tts\":{\"language\":\"en\",\"rules\":{\"critical_hp\":{\"tank\":0,\"heal\":0,\"dps\":0,\"all\":0},\"critical_mp\":{\"tank\":0,\"heal\":0,\"dps\":0,\"all\":0},\"top\":{\"dps\":false,\"hps\":false,\"tps\":false},\"aggro\":false,\"encounter\":{\"start\":false,\"end\":false}}},\"table_settings\":{\"general\":{\"table\":{\"short_names\":\"no_short\",\"footer_at_top\":true,\"percent_bars\":true,\"prioritize_party\":false},\"raid\":{\"short_names\":\"no_short\",\"percent_bars\":true,\"prioritize_party\":false}},\"dps\":{\"show_footer\":false},\"heal\":{\"show_footer\":false},\"tank\":{\"show_footer\":false}},\"table_columns\":{\"dps\":[\"damage%\",\"encdps\",\"crithit%\",\"DirectHitPct\",\"CritDirectHitPct\"],\"heal\":[\"healed%\",\"effective_heal_pct\",\"enchps\",\"effective_hps\",\"OverHealPct\",\"critheal%\"],\"tank\":[\"damage_taken_pct\",\"enctps\",\"damagetaken\",\"healstaken\",\"BlockPct\",\"ParryPct\",\"deaths\"],\"raid\":{\"dps\":[\"encdps\",\"enchps\"],\"heal\":[\"enchps\",\"encdps\"],\"tank\":[\"encdps\",\"enchps\"]}},\"sort_columns\":{\"dps\":\"damage\",\"heal\":\"healed\",\"tank\":\"damagetaken\",\"raid\":\"damage\"},\"detail_data\":{\"dps\":[\"damage%\",\"encdps\",\"damage\",\"crithit%\",\"DirectHitPct\",\"CritDirectHitPct\",\"max_hit_format\"],\"heal\":[\"healed%\",\"effective_heal_pct\",\"enchps\",\"effective_hps\",\"healed\",\"OverHealPct\",\"critheal%\",\"max_heal_format\"],\"tank\":[\"damage_taken_pct\",\"damagetaken\",\"enctps\",\"healstaken\",\"BlockPct\",\"deaths\"]},\"discord\":{\"url\":\"\",\"metrics\":[\"encdps\",\"enchps\"],\"sort\":\"damage\"},\"spells_mode\":{\"spells\":[\"7443\"],\"effects\":[],\"dots\":[\"163\",\"1871\",\"2616\",\"1881\"],\"debuffs\":[\"2586\",\"3667\"],\"party_spells\":[],\"party_effects\":[],\"party_dots\":[],\"party_debuffs\":[],\"reverse_skill\":false,\"reverse_effect\":false,\"reverse_dot\":false,\"reverse_debuff\":false,\"party_reverse_skill\":false,\"party_reverse_effect\":false,\"party_reverse_dot\":false,\"party_reverse_debuff\":false,\"always_skill\":true,\"always_effect\":false,\"always_dot\":true,\"always_debuff\":true,\"warning_threshold\":6,\"spells_per_row\":1,\"show_icon\":true,\"use_tts\":false,\"party_use_tts\":false,\"tts_on_effect\":false,\"party_tts_on_skill\":false,\"party_tts_on_effect\":false,\"party_zones\":[],\"tts_trigger\":\"zero\",\"minimal_layout\":false,\"invert_vertical\":false,\"invert_horizontal\":false,\"designer\":{\"skill\":{\"warning\":true,\"indicator\":\"ticking\",\"cooldown_bottom_left\":false},\"effect\":{\"border\":false,\"warning\":true,\"indicator\":\"ticking\",\"cooldown_bottom_left\":false},\"dot\":{\"border\":true,\"warning\":true,\"indicator\":\"ticking\",\"cooldown_bottom_left\":false},\"debuff\":{\"border\":false,\"warning\":true,\"indicator\":\"ticking\",\"cooldown_bottom_left\":false},\"general\":{\"show_hover_names\":false}},\"ui\":{\"use\":false,\"sections\":{}},\"layout\":\"normal\",\"always_dot_static\":true,\"always_debuff_static\":true}}"
```

**Comment by GoldenChrysus** [comment]
Resolved in version 1.9.3. Please check.

**Comment by Squishums** [comment]
Confirmed fixed in the latest version. Thanks for the quick update.
